module/zcommon/zfeature_common.c

```diff
@@ -137,6 +158,33 @@ deps_contains_feature(const spa_feature_t *deps, const spa_feature_t feature)
 	return (B_FALSE);
 }
 
+static boolean_t
+zfs_mod_supported_feature(const char *name)
+{
+/*
+ * The zfs module spa_feature_table[], whether in-kernel or in libzpool,
+ * always supports all the features. libzfs needs to query the running
+ * module, via sysfs, to determine which features are supported.
+ */
+#if defined(_KERNEL) || defined(LIB_ZPOOL_BUILD)
+	return (B_TRUE);
+#else
+	struct stat64 statbuf;
+	char *path;
+	boolean_t supported = B_FALSE;
+	int len;
+
+	len = asprintf(&path, "%s/%s/%s", ZFS_SYSFS_DIR,
+	    ZFS_SYSFS_POOL_FEATURES, name);
+
+	if (len > 0) {
+		supported = stat64(path, &statbuf) == 0;
```

Comment:
nit: personally I find using `!!` for this more readable.  But whichever you prefer is OK by me.

Reply:
fixed